src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId || tabs[0]);
```

Comment:
There is a potential issue with the default active tab logic. The expression `tabs.find(tab => tab.id === activeTabId || tabs[0])` will not correctly default to the first tab if no active tab is found. Instead, it will return `tabs[0]` if `activeTabId` is falsy, which is not the intended behavior. Consider using `tabs.find(tab => tab.id === activeTabId) || tabs[0]` to ensure the first tab is used as a default when no match is found.